packages/firestore/test/unit/specs/describe_spec.ts

```diff
@@ -41,14 +41,12 @@ const KNOWN_TAGS = [
   NO_IOS_TAG
 ];
 
-// Filters out multi-client tests if persistence is not enabled.
-const MULTI_CLIENT_TEST_FILTER = (
-  tags: string[],
-  persistenceEnabled: boolean
-) => tags.indexOf(MULTI_CLIENT_TAG) !== -1 && persistenceEnabled;
-
-const WEB_SPEC_TEST_FILTER = (tags: string[]) =>
-  tags.indexOf(NO_WEB_TAG) === -1;
+const WEB_SPEC_TEST_FILTER = (tags: string[], persistenceEnabled: boolean) => {
+  return (
+    tags.indexOf(NO_WEB_TAG) === -1 &&
+    (tags.indexOf(MULTI_CLIENT_TAG) === -1 || persistenceEnabled)
```

Comment:
Seeing this code now it looked incorrect that we don't check for PERSISTENCE_TAG too, and in fact we don't check it anywhere, and it doesn't exist on iOS or Android either. I think it was a temporary measure that we used before all the platforms had persistence.  Can you remove the tag here (and its usage as 'persistence' in persistence_spec.test.ts)?  Else I can do it against master, but it'll probably cause a conflict with this PR.